extensions/standard-processors/processors/GenerateFlowFile.cpp

```diff
@@ -61,6 +62,13 @@ core::Property GenerateFlowFile::UniqueFlowFiles(
     core::PropertyBuilder::createProperty("Unique FlowFiles")->withDescription("If true, each FlowFile that is generated will be unique. If false, a random value will be generated and all FlowFiles")
         ->isRequired(false)->withDefaultValue<bool>(true)->build());
 
+core::Property GenerateFlowFile::CustomText(
+    core::PropertyBuilder::createProperty("Custom Text")
+      ->withDescription("If Data Format is text and if Unique FlowFiles is false, then this custom text will be used as content of the generated FlowFiles and the File Size will be ignored. "
+                        "Finally, if Expression Language is used, evaluation will be performed only once per batch of generated FlowFiles")
+      ->supportsExpressionLanguage(true)
```

Comment:
we could emit a warning when Custom Text is set but it won't be used

Reply:
+1

Reply:
Added warning and test in 8b3702dd930f7b38cc744e3e1917b6c7380816b8